lib/reports/ReportPeakHourFactor.js

```diff
@@ -0,0 +1,323 @@
+/* eslint-disable class-methods-use-this */
+import ArrayUtils from '@/lib/ArrayUtils';
+import { ReportBlock, ReportType } from '@/lib/Constants';
+import ReportBaseFlow from '@/lib/reports/ReportBaseFlow';
+import ReportBaseFlowDirectional from '@/lib/reports/ReportBaseFlowDirectional';
+import ReportCountSummaryTurningMovement from '@/lib/reports/ReportCountSummaryTurningMovement';
+import {
+  indexRangeAm,
+  indexRangeMax,
+  indexRangePeakHour,
+  indexRangePm,
+  indexRangeSum,
+} from '@/lib/reports/time/ReportTimeUtils';
+import TimeFormatters from '@/lib/time/TimeFormatters';
+
+const dirs = ['N', 'E', 'S', 'W'];
```

Comment:
we should ideally use `CardinalDirection.enumValues.map(({ short }) => short)` and `TurningMovement.enumValues.map(({ short }) => short)` here.  (same for a few of the other modules that hardcode these - it's easier if we can unify this across all report types!)